docs/guides/using_filters.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: home
+title: Using filters to narrow your search
+parent: How-to guides
+nav_order: 5
+---
+
+## Using filters to narrow your search
+
+TIMDEX includes a set of filters that you can use to limit your search. The [API reference](https://mitlibraries.github.io/timdex/reference/#query-search)
+for the search operation lists each available filter and how to use them. In this how-guide, we'll look at the
+sourceFilter, but the process is similar with other filters. If you're not sure what values a filter will accept, try
+a search that [returns aggregations](viewing_aggregations).
+
+This example demonstrates a query that searches for the term "data" in the "dspace@mit" or the "abdul latif jameel
```

Comment:
We aren't maintaining daily updates at all for any of the RDI sources. The only sources we will be promoting as available are: dspace, aspace, alma/catalog. I'll share more info on this in Slack to make sure we're on the same page about this as I suspect both that we haven't communicated this well _and_ we may actually need to confirm what the APIs do by default matches our intent.